rule.go

```diff
@@ -1,11 +1,8 @@
 /* Copyright 2016 Google Inc.
-
```

Comment:
why are these lines being modified?
please remove these added '-' in the license.

Reply:
rather i misread this, please reinsert these blank lines.

Reply:
I must have some weird vs code configuration, done!